Dangerfile

```diff
@@ -23,6 +23,13 @@ Please run `bundle install` to make sure they match.
   fail(message)
 end
 
+# Check that the PR contains changes to the CHANGELOG.md file.
+#  - If it's a feature PR, CHANGELOG should have a new entry describing the changes
+#  - If it's a release PR, we expect the CHANGELOG to have been updated during `rake new_release` with updated section title + new placeholder section
+unless git.modified_files.include?('CHANGELOG.md')
+  warn 'Please add an entry in the CHANGELOG.md file to describe the changes made by this PR'
```

Comment:
Note that in this current PR, we still get the warning because the file was just introduced so it's in `git.added_files` not in `git.modified_files`. Not worth adding a case for this in the Danger rule since this is the only PR where this will happen, for the original introduction of the file.